src/reliability.jl

```diff
@@ -0,0 +1,72 @@
+using LinearAlgebra: Diagonal
+
+struct Reliability
+    alpha::Float64
+    dropped::Vector{Pair{Int, Float64}}
+end
+
+function Base.show(io::IO, x::Reliability)
+    println(io, "Reliability for all items: $(round(x.alpha; digits=3))")
+    println(io, "")
+    println(io, "Reliability if an item is dropped:")
+    for i ∈ x.dropped
+        println(io, "item $(i.first): $(round(i.second; digits=3))")
+    end
+end
+
+function _alpha(covmatrix::AbstractMatrix{T}) where T <: Real
+    k = size(covmatrix, 2)
```

Comment:
I would check if the matrix is square and symmetric (approximately).

Reply:
and with non-negative entries.
I assume that checking if it is a covariance matrix is an overkill.

Reply:
and that `k>1`.

Reply:
Something like:

```julia
using LinearAlgebra: cholesky

@assert issuccess(cholesky(cox_matrix, Val(true)))
```